test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+describe RecipesController do
+  describe "search" do
+    it "responds with success" do
+      get root_path
+
+      must_respond_with :success
+    end
+  end
+
+  describe "index" do
+    it "responds with success for real search" do
+      VCR.use_cassette("recipes") do
+        query = "chicken"
+        get results_path, params: {query: query}
+      end
+
+      must_respond_with :success
+    end
+
+    it "responds with success for search with no results" do
+      VCR.use_cassette("recipes") do
+        query = "something that doesnt exist"
+        get results_path, params: {query: query}
```

Comment:
Good job catching this error case!

Reply:
It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?